po/fa.po

```diff
@@ -46,61 +46,60 @@ msgstr "نمونه کد"
 
 #: src/SUMMARY.md
 msgid "Running Cargo Locally"
-msgstr "اجرای cargo روی ماشین local"
+msgstr "اجرای کد به صورت لوکال با Cargo"
 
 #: src/SUMMARY.md
 msgid "Day 1: Morning"
 msgstr "روز اول: صبح"
 
-#: src/SUMMARY.md
+#: src/SUMMARY.md src/running-the-course/course-structure.md
+#: src/welcome-day-1.md src/welcome-day-2.md src/welcome-day-3.md
+#: src/welcome-day-4.md src/concurrency/welcome-async.md
 msgid "Welcome"
-msgstr "خوش آمدید"
+msgstr "خوش آمد"
 
-#: src/SUMMARY.md src/hello-world.md src/hello-world/hello-world.md
-#, fuzzy
+#: src/SUMMARY.md src/running-the-course/course-structure.md
+#: src/welcome-day-1.md src/hello-world.md src/types-and-values.md
+#: src/types-and-values/hello-world.md
 msgid "Hello, World"
-msgstr "سلام دنیا"
+msgstr "سلام, دنیا"
 
-#: src/SUMMARY.md src/hello-world/what-is-rust.md
+#: src/SUMMARY.md src/hello-world.md src/hello-world/what-is-rust.md
 msgid "What is Rust?"
 msgstr "زبان Rust چیست؟"
 
-#: src/SUMMARY.md src/hello-world/benefits.md
+#: src/SUMMARY.md src/hello-world.md src/hello-world/benefits.md
 msgid "Benefits of Rust"
-msgstr ""
+msgstr "مزیت های rust"
 
-#: src/SUMMARY.md src/hello-world/playground.md
+#: src/SUMMARY.md src/hello-world.md src/hello-world/playground.md
 msgid "Playground"
 msgstr ""
 
-#: src/SUMMARY.md src/types-and-values.md
+#: src/SUMMARY.md src/running-the-course/course-structure.md
+#: src/welcome-day-1.md src/types-and-values.md
 msgid "Types and Values"
-msgstr ""
+msgstr "تایپ . مقدار ها"
 
-#: src/SUMMARY.md src/types-and-values/variables.md
+#: src/SUMMARY.md src/types-and-values.md src/types-and-values/variables.md
 msgid "Variables"
 msgstr "متغیرها"
 
-#: src/SUMMARY.md src/types-and-values/values.md
+#: src/SUMMARY.md src/types-and-values.md src/types-and-values/values.md
 msgid "Values"
-msgstr ""
+msgstr "مقدار ها"
 
-#: src/SUMMARY.md src/types-and-values/arithmetic.md
+#: src/SUMMARY.md src/types-and-values.md src/types-and-values/arithmetic.md
 msgid "Arithmetic"
-msgstr ""
-
-#: src/SUMMARY.md src/types-and-values/strings.md
-msgid "Strings"
```

Comment:
Oh, so as @mgeisler said these are because of the updated text. I generated the `messages.pot` and that is the case.

I'll resolve this one, although it was better if this was done in an other separate merge requests.